pk-library/src/main/java/me/brendanweinstein/views/CVVEditText.java

```diff
@@ -40,6 +42,13 @@ private void setup() {
 	public void setCardEntryListener(CardEntryListener listener) {
 		mListener = listener;
 	}
+
+	public void setCvvMaxLength(int cvvMaxLength) {
+        mMaxLength = cvvMaxLength;
+        InputFilter[] filters = new InputFilter[1];
+        filters[0] = new InputFilter.LengthFilter(mMaxLength);
+        setFilters(filters);
+    }
```

Comment:
Bummer, I think this lib is using tabs instead of spaces. Might be nice to conform in case we ever upstream it